.github/workflows/ci.yml

```diff
@@ -78,12 +78,17 @@ jobs:
       - uses: ruby/setup-ruby@v1
         with:
           bundler-cache: true
+          ruby-version: 2.7
+          working-directory: standards-catalogue
 
       - name: Download built site
         uses: actions/download-artifact@v2
         with:
           name: site
-          path: build
+          path: built-site
```

Comment:
Does this path need to match with what we're using in the `upload-artifact` part?

Reply:
ah sorry, this is the path it's downloaded to - ignore me